### ISSUE TITLE
Mario's textures are inverted

### ISSUE THREAD

**Issue by ianfrancong** [headshot2017/sm64-san-andreas#33]
![20241216185200_1](https://github.com/user-attachments/assets/88b79014-c424-4f1d-a029-ef54b934aa8d)
I am using several mods which may be the cause of this, but I don't know which one could be the culprit.

**Comment by headshot2017** [comment]
do you have skygfx installed?

**Comment by ianfrancong** [comment]
Yeah, I removed the mod and that fixed the issue. Still a bummer that it isn't compatible with skygfx

**Comment by headshot2017** [comment]
it actually IS compatible but you need to change some settings. this is from this mod's readme:

Known issues

Using this with the SkyGFX mod makes Mario extremely bright, and the textures in his face (eyes, moustache, hat logo...) don't render correctly.

Disable "dualPass" in all the skygfx .ini files (change 1 to 0) to fix this issue.

**Comment by ianfrancong** [comment]
Thanks
